tests/web/test_helpers.py

```diff
@@ -0,0 +1,42 @@
+from unittest.mock import MagicMock
+
+import auslib.web.public.helpers
+import auslib.web.public.json
+
+
+def test_get_content_signature_headers(monkeypatch):
+    mockapp = MagicMock()
+    mockapp.config = {
+        "AUTOGRAPH_product_URL": "AUTOGRAPH_product_URL",
```

Comment:
Just for readability, can you change the values here to not match the keys, please? I spent a good few minutes wondering why `mock_sign_hash` was looking at keys before realizing it was actually receiving the values :laughing:

Reply:
Oops :) That was indeed confusing. Fixed!